_posts/2020-08-07-g2p.md

```diff
@@ -148,11 +152,63 @@ mappings:
     mapping: eng_ipa_to_arpabet.json 
 ```
 
-### Special settings
+If you're not familiar with yaml, and you're not just copy pasting from here, I recommend having a look at one of the many [tutorials](https://gettaurus.org/docs/YAMLTutorial/) on how to use yaml properly before attempting to write your own mapping configuration.
+
+====
+***PROPOSED END OF FIRST POST***
```

Comment:
What's missing here and should be in the first post is where the heck are you supposed to place these files to have them used. It's in the later blog posts, but is there a way to make this first post a usable minimal working example? If not, put a forward pointer saying how to run these will come in a later post.

Reply:
Good point, I guess this is a reflection of me waffling a bit about whether I should include any of the yaml stuff in the first post at all. It might be easier to make the first post almost exclusively about writing rules in the G2P Studio. Then, a separate basic blog post for writing rules but on your computer with a clone of `g2p`, and then the other two blog posts later.

Reply:
I like that idea. If you move the YAML discussion off to where you talk about working in a clone, then it makes sense. In the current version, the YAML stuff seems to come out of nowhere and it's not clear what to do with it, but in that alternate version it would hang together. That's my vote, if you're willing to do that extra work...